### ISSUE TITLE
podman-compose down prevents systemd service from starting due to missing CIDFile

### ISSUE THREAD

**Issue by runbmp** [containers/podman-compose#494]
**Describe the bug**
Generating systemd units from a podman-compose container does not work after taking down the container.  See the sequence of trouble commands:
```
podman-compose up -d
podman generate systemd --new --name --files photoprism
mv container-photoprism.service ~/.config/systemd/user
systemctl --user enable --now container-photoprism.service
```
then you end up with 
```
May 06 10:28:39 600sff podman[3378509]: Error: error reading CIDFile: open /run/user/1000/container-photoprism.service.ctr-id: no such file >
May 06 10:28:39 600sff systemd[1046]: container-photoprism.service: Control process exited, code=exited, status=125/n/a
░░ Subject: Unit process exited
░░ Defined-By: systemd
░░ Support: https://lists.freedesktop.org/mailman/listinfo/systemd-devel
░░
░░ An ExecStopPost= process belonging to unit UNIT has exited.
░░
░░ The process' exit code is 'exited' and its exit status is 125.
```
This could very well be user error, but why does the systemd unit require some remnant from `podman-compose` to work.  If I use `podman-compose stop` instead of `podman-compose down` I do not have this issue.

**Comment by mokibit** [comment]
Closing this issue as it is no longer relevant:
- `podman generate systemd` is deprecated - Podman now recommends using Quadlets or the built-in `podman-compose systemd` command.
- `podman-compose` has its own systemd integration that operates at the pod level and correctly handles `podman-compose down` followed by restarts.

New `podman-compose systemd`:
```
cat /etc/systemd/user/podman-compose@.service
```
uses `ExecStartPre=-podman-compose up --no-start` which recreates the pod and containers before starting.

Here is a minimal example showing the correct workflow that works:
```
Debian GNU/Linux 13 (trixie)
Python 3.13.5
podman-compose version 1.6.0
podman version 5.4.2
```
`docker-compose.yml` file:
```
services:
  testbox:
    image: busybox
    command: ["sleep", "3600"]
```
Register the project with systemd:
```
$ podman-compose systemd -a register
```
Create the systemd unit template:
```
$ podman-compose systemd -a create-unit
$ systemctl --user daemon-reload
```
Start the service:
```
$ systemctl --user start podman-compose@<project_name>
```
Verify it's running:
```
$ podman ps --filter "name=<project_name>_testbox_1"
```
Run podman-compose down (the problematic command from the original issue):
```
$ podman-compose down
```
Restart the service - this works correctly:
```
systemctl --user start podman-compose@<project_name>
```
Expected success: The service starts and the container is running.
The recommended `podman-compose systemd` workflow correctly handles `podman-compose down` and subsequent restarts.


If you must use `podman generate systemd` (despite deprecation), use `podman-compose stop` instead of `podman-compose down` to avoid removing the container and pod.
